packages/data/src/plugins/persistence/index.js

```diff
@@ -427,24 +434,29 @@ persistencePlugin.__unstableMigrate = ( pluginOptions ) => {
 	// Other ad-hoc preferences.
 	migrateIndividualPreferenceToPreferencesStore(
 		persistence,
-		'core/edit-post',
+		{ from: 'core/edit-post', scope: 'core/edit-post' },
 		'hiddenBlockTypes'
 	);
 	migrateIndividualPreferenceToPreferencesStore(
 		persistence,
-		'core/edit-post',
+		{ from: 'core/edit-post', scope: 'core/edit-post' },
 		'editorMode'
 	);
 	migrateIndividualPreferenceToPreferencesStore(
 		persistence,
-		'core/edit-post',
+		{ from: 'core/edit-post', scope: 'core/edit-post' },
 		'preferredStyleVariations'
 	);
 	migrateIndividualPreferenceToPreferencesStore(
 		persistence,
-		'core/edit-site',
+		{ from: 'core/edit-site', scope: 'core/edit-site' },
 		'editorMode'
 	);
+	migrateIndividualPreferenceToPreferencesStore(
+		persistence,
+		{ from: 'core/block-editor', scope: 'core' },
```

Comment:
I don't know that there's any advantage to grouping preferences per package in that way now.

The advantage of using 'core' is that if the feature is moved between packages, the scope still makes sense and a data migration isn't needed.

I think it'll be good to use 'core' for anything shared across editors in this way.

I should update the docs to make this usage clear. I'll work on a separate PR for that.

Reply:
> The advantage of using 'core' is that if the feature is moved between packages, the scope still makes sense and a data migration isn't needed.

Not sure I understand. Would we be moving it from preferences to somewhere else?

> I think it'll be good to use 'core' for anything shared across editors in this way.

Do we really want the usage to be shared across editors though? It's unlikely that most used/most relevant blocks in the post editor will be the same as in the site editor.

Reply:
> Not sure I understand. Would we be moving it from preferences to somewhere else?

It's not so much the preferences, but the code that interacts with the preferences store could be moved. As an example, the code that tracks `insertUsage` has already been moved in the past from the `editor` package to `block-editor`. If it moved from `block-editor` to elsewhere again in the future it's better not to have to change the scope.

The scope is more related to how widely the data is used rather than where the feature is implemented.

> Do we really want the usage to be shared across editors though? It's unlikely that most used/most relevant blocks in the post editor will be the same as in the site editor.

It's already shared across editors. It's a good point, that could be changed, but not something I'd tackle in this PR.

Reply:
> If it moved from block-editor to elsewhere again in the future it's better not to have to change the scope.

Got it, that makes sense.